projects/m1/005-units-of-time-again/js/index.js

```diff
@@ -0,0 +1,44 @@
+// inserimento input utente
+let totalInputSeconds = 673876543;
+
+// dichiarazioni variabili
+let days = 86400;
+let hours = 3600;
+let minutes = 60
```

Comment:
It is recommended to see:
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/